packages/compass-indexes/src/components/create-index-modal/create-index-modal.tsx

```diff
@@ -103,7 +103,7 @@ function CreateIndexModal({
       className={modalStyles}
       contentClassName={modalContentWrapperStyles}
     >
-      <Body className={modalContentStyles}>
+      <Body as="div" className={modalContentStyles}>
```

Comment:
There are tons of errors because <Body> makes a <p> by default and you can't put fieldsets and things inside that. Just making it a div seems to help.

Reply:
Tbh this would be a much better default, maybe this is something we can fix with our re-export from compass-components

Reply:
As discussed I think I'll leave this for now. Some places probably depend on it being a <p> and this is unlikely to be the only place with nested Body components..